src/components/ProductCategoriesDialog.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-dialog>
+    <v-card>
+      <v-card-title>
+        {{ $t('ProductCategories.Title') }} <v-btn style="float:right;" variant="text" density="compact" icon="mdi-close" @click="close"></v-btn>
+      </v-card-title>
+
+      <v-divider></v-divider>
+
+      <v-card-text v-if="categories.length">
+        <v-chip v-for="category in categories" :key="category" label class="mr-2 mb-2">
+          {{ category }}
+        </v-chip>
+      </v-card-text>
+      <v-card-text v-if="!categories.length">
+        <span class="text-red">{{ $t('ProductCategories.Empty') }}</span>
```

Comment:
Link to the OFF product page maybe?

Reply:
yeah we should ! will add in another PR (currently the `ProductCategoriesDialog` component doesn't have any info about the product, need to change it)